superduperdb/backends/mongodb/datastore.py

```diff
@@ -7,15 +7,15 @@
 import pymongo
 
 from superduperdb import logging
-from superduperdb.backends.base.data_backend import BaseDataBackend
+from superduperdb.backends.base.datastore import DataStore
 from superduperdb.backends.mongodb.artifacts import MongoArtifactStore
 from superduperdb.backends.mongodb.metadata import MongoMetaDataStore
 from superduperdb.base.serializable import Serializable
 from superduperdb.misc.colors import Colors
 from superduperdb.misc.special_dicts import MongoStyleDict
 
 
-class MongoDataBackend(BaseDataBackend):
+class MongoDataBackend(DataStore):
```

Comment:
if we are making BaseDataBackend -> DataStore 
we should have MongoDataBackend -> MongoDataStore

Reply:
I actually like `DataBackend`, since it allows us to do things like having a directory of `.csv` files, for instance.